### PR TITLE
Post dev

### PR DESCRIPTION
**Pull request by zhoudian64**
✨add POST CRUD
✨add REPLY CRUD
❓决定删除POST和REPLY的用户权限。

### CODE REVIEW

**Comment by zhoudian64** [comment]
@AdrianDuan plz review

File: services/post.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		Error
+
+	if err != nil {
+		return
+	}
+
+	err = db.
+		Preload("Creator", func(pg *gorm.DB) *gorm.DB {
+			return pg.Select("id, name, avatar, state, roles")
+		}).
```
Comment (AdrianDuan) [inline services/post.go:48]: Remove "state" and "roles" fields here, and add "username" field

File: services/post.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+		Error
+
+	return
+}
+
+// GetPost get post by id
+func (p PostService) GetPost(id string) (post models.Post, err error) {
+	err = p.PG.
+		Preload("Creator").
+		Where("id = ?", id).
```
Comment (AdrianDuan) [inline services/post.go]: Only select "id", "name", "username" and "avatar" fields

**Comment by AdrianDuan** [comment]
Please help to add xss filter for all "content" field in post entity and reply entity

**Pull request closed without merging**